### PR TITLE
Riley McCullough

### PR DESCRIPTION
**Pull request by Rileybmcc**
Resubmitting my Mod 0 project!

### CODE REVIEW

File: final_prep/README.md [hunk 1]
```diff
@@ -44,31 +44,34 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1440" alt="Screen Shot 2022-04-10 at 8 40 04 PM" src="https://user-images.githubusercontent.com/102133027/162656941-bd565e60-0053-4c64-9274-c4bfcb17f78e.png">
+
+- <img width="1440" alt="Screen Shot 2022-04-10 at 8 44 22 PM" src="https://user-images.githubusercontent.com/102133027/162656959-a9701196-420d-4046-96da-85ed462a6fd3.png">
```
Comment (ameseee) [inline final_prep/README.md:49]: Looks like a good amount of outside-of-class Turing time is there, as well as those workouts - if that's why you need, GREAT!

File: final_prep/README.md [hunk 2]
```diff
@@ -44,31 +44,34 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
...
+
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: https://gist.github.com/Rileybmcc/f0d3c2f7975dbe8a059f8a3a535fcf39
```
Comment (ameseee) [inline final_prep/README.md:60]: Great intro message

**Review by ameseee — COMMENTED** [review]
HI @Rileybmcc this looks like a solid project - you are going to Mod 1! Thank you!

**Comment by Rileybmcc** [comment]
Thank you so much! I was very worried you'd ask me to repeat after all my
submission troubles!
-Riley

**Pull request closed without merging**